backend/convert_from_protobuf.go

```diff
@@ -89,6 +91,7 @@ func (f ConvertFromProtobuf) PluginContext(proto *pluginv2.PluginContext) Plugin
 		OrgID:                      proto.OrgId,
 		PluginID:                   proto.PluginId,
 		PluginVersion:              proto.PluginVersion,
+		APIVersion:                 proto.ApiVersion,
```

Comment:
Nit. Do we want to enforce any default value when converting from/to proto in case it's empty?

Reply:
I would say no: It can be useful to identify plugins that define an API version vs not